Sources/Classes/core/JavascriptBridge/MiniAppScriptMessageHandler.swift

```diff
@@ -745,6 +747,37 @@ internal class MiniAppScriptMessageHandler: NSObject, WKScriptMessageHandler {
     }
 }
 
+//MARK: Universal Bridge support
+extension MiniAppScriptMessageHandler {
+    func sendJsonToHostApp(requestParam: RequestParameters?, callbackId: String) {
+        guard let requestParamValue = requestParam?.jsonInfo else {
+            executeJavaScriptCallback(responseStatus: .onError, messageId: callbackId, response: MiniAppJavaScriptError.invalidPermissionType.rawValue)
+            return
+        }
+        if !requestParamValue.content.isEmpty {
+            let info = MiniAppBridgeContent(jsonStringContent: requestParamValue.content)
+            self.hostAppMessageDelegate?.sendJsonToHostApp(info: info, completionHandler: { (result) in
+                self.manageJsonStringToHostAppResult(result, with: callbackId)
+            })
+        } else {
+            self.executeJavaScriptCallback(responseStatus: .onError, messageId: callbackId, response: getMiniAppErrorMessage(MiniAppJavaScriptError.valueIsEmpty))
```

Comment:
```suggestion
            self.executeJavaScriptCallback(responseStatus: .onError, messageId: callbackId, response: prepareMAJavascriptError(MiniAppJavaScriptError.valueIsEmpty))
```

Reply:
This is updated in the latest commit.

Reply:
I'm sorry. I don't see this change yet.

Reply:
My Bad it was missed in the previous commit will be changed in latest commit.